components/infobox/wikis/ageofempires/infobox_person_player_custom.lua

```diff
@@ -416,4 +417,12 @@ function CustomPlayer._getBroadcastGames()
 	return games
 end
 
+function CustomPlayer._getPersonQuery()
+	if Namespace.isMain() then
+		return _player.pagename
+	else
+		return mw.ext.TeamLiquidIntegration.resolve_redirect(_args.id)
```

Comment:
not sure this is a good idea even in non main space
you might have players with link =/= id even there
maybe subpagename (`mw.title.getCurrentTitle().subpageText`) or the last titlepart would be a better option

Reply:
Also i don't really care about userspace. If the ID is functional to query, i'm fine with it - the ID has a higher chance to be correct than the pagename/titlepart.

Reply:
alright^^